applications/frontier-pos-template.md

```diff
@@ -0,0 +1,184 @@
+# frontier-pos-template
+
+- **Team Name:** ChainSupport
+- **Payment Details:**
+  - **Payment**: 0xcC1974d462B9F22F37D09c70F6023Ef9167EB3De (USDC)
```

Comment:
We need a Polkadot and AssetHub address. Payments on Ethereum are not offered anymore.